index.html

```diff
@@ -60,5 +61,387 @@ <h2>I am a Software Developer</h2>
+              </a>
+            </div>
+            <div class="gridItem grid8 backGroundImg">
+              <h2>
+                Profesional Art<br />
+                Printing Data
+              </h2>
+              <p>
+                A daily selection of privately<br />
+                personalized reads; no accounts or<br />
+                sign-ups required. has been the<br />
+                industry's standard
+              </p>
+              <ul class="d-flex">
+                <li>
+                  <a href="#">
+                    <img
+                      alt="htmlButton"
+                      src="img/htmlTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      alt="bootstrapButton"
+                      src="img/bootstrapTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      alt="rubyButton"
+                      src="img/rubyTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+              </ul>
+              <a href="#">
+                <img
+                  alt="seeProjectButton"
+                  src="img/Action.png"
+                  class="buttonProperties"
+                />
+              </a>
+            </div>
+          </div>
+        </section>
+        <section class="aboutMe">
+          <div class="d-flex bottom flex-center">
+            <h2>About me</h2>
+            <p>
+              Hello I’m a software developer! I can help<br />
+              you build a product, feature or website<br />
+              Look through some of my work and<br />
+              experience! If you like what you see and<br />
+              have a project you need coded, don’t<br />
+              hestiate to contact me.
+            </p>
+            <a href="#">
+              <img alt="resume" src="img/Resume.png" />
+            </a>
+          </div>
+          <img alt="division" src="img/line2.png" class="line2" />
+        </section>
+        <section class="skills d-flex">
+          <ul>
+            <li>
+              <h2 class="lan">Language</h2>
+              <ul class="eniimi">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+              </ul>
+            </li>
+            <li>
+              <h2 class="frame">Frameworks</h2>
+              <ul class="eniimi">
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
```

Comment:
- [ ] Please also notice on the headings for each of these skills. the space between the text and the symbol on the left should be much greater.
- [ ] Please address this issue on the other Skills as well. that is the "Languages" & "Skills" headings.

|Your Design | Reference Design |
-|-
|![image](https://user-images.githubusercontent.com/18730106/153727147-37386222-90eb-4ff1-a1b3-2bf03293848c.png)|![image](https://user-images.githubusercontent.com/18730106/153727151-722ea018-4935-4a39-a538-10f216ccbfbe.png)|